### ISSUE TITLE
(ES Locale) TextField long press copy-paste: 'pasteButtonLabel' was called on null

### ISSUE THREAD

**Issue by xascis** [flutter/flutter#24599]
## Steps to Reproduce

1. Create an app with Spanish locale
```
      localizationsDelegates: [
        GlobalMaterialLocalizations.delegate,
        GlobalWidgetsLocalizations.delegate,
      ],
      supportedLocales: [
        const Locale('en', 'EN'),
        const Locale('es', 'ES'),
      ],
```
2. Run the app on iOS device / simulator configured in Spanish language
3. Long tab a textfield

If I remove 'const Locale('es', 'ES'),' works fine.

## Logs

```
flutter: ══╡ EXCEPTION CAUGHT BY WIDGETS LIBRARY ╞═══════════════════════════════════════════════════════════
flutter: The following NoSuchMethodError was thrown building _TextSelectionToolbar(dirty):
flutter: The getter 'pasteButtonLabel' was called on null.
flutter: Receiver: null
flutter: Tried calling: pasteButtonLabel
flutter:
flutter: When the exception was thrown, this was the stack:
flutter: #0      Object.noSuchMethod (dart:core/runtime/libobject_patch.dart:50:5)
flutter: #1      _TextSelectionToolbar.build (package:flutter/src/cupertino/text_selection.dart:94:51)
flutter: #2      StatelessElement.build (package:flutter/src/widgets/framework.dart:3752:28)
flutter: #3      ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3699:15)
flutter: #4      Element.rebuild (package:flutter/src/widgets/framework.dart:3547:5)
flutter: #5      ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3679:5)
flutter: #6      ComponentElement.mount (package:flutter/src/widgets/framework.dart:3674:5)
flutter: #7      Element.inflateWidget (package:flutter/src/widgets/framework.dart:2950:14)
flutter: #8      Element.updateChild (package:flutter/src/widgets/framework.dart:2753:12)
flutter: #9      SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:4838:14)
flutter: #10     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2950:14)
flutter: #11     Element.updateChild (package:flutter/src/widgets/framework.dart:2753:12)
flutter: #12     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:4838:14)
flutter: #13     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2950:14)
flutter: #14     Element.updateChild (package:flutter/src/widgets/framework.dart:2753:12)
flutter: #15     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:4838:14)
flutter: #16     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2950:14)
flutter: #17     Element.updateChild (package:flutter/src/widgets/framework.dart:2753:12)
flutter: #18     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:4838:14)
flutter: #19     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2950:14)
flutter: #20     Element.updateChild (package:flutter/src/widgets/framework.dart:2753:12)
flutter: #21     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3710:16)
flutter: #22     Element.rebuild (package:flutter/src/widgets/framework.dart:3547:5)
flutter: #23     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3679:5)
flutter: #24     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:3826:11)
flutter: #25     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3674:5)
flutter: #26     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2950:14)
flutter: #27     Element.updateChild (package:flutter/src/widgets/framework.dart:2753:12)
flutter: #28     RenderObjectElement.updateChildren (package:flutter/src/widgets/framework.dart:4621:32)
flutter: #29     MultiChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:4953:17)
flutter: #30     Element.updateChild (package:flutter/src/widgets/framework.dart:2742:15)
flutter: #31     _TheatreElement.update (package:flutter/src/widgets/overlay.dart:507:16)
flutter: #32     Element.updateChild (package:flutter/src/widgets/framework.dart:2742:15)
flutter: #33     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3710:16)
flutter: #34     Element.rebuild (package:flutter/src/widgets/framework.dart:3547:5)
flutter: #35     BuildOwner.buildScope (package:flutter/src/widgets/framework.dart:2286:33)
flutter: #36     _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding&PaintingBinding&SemanticsBinding&RendererBinding&WidgetsBinding.drawFrame (package:flutter/src/widgets/binding.dart:676:20)
flutter: #37     _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding&PaintingBinding&SemanticsBinding&RendererBinding._handlePersistentFrameCallback (package:flutter/src/rendering/binding.dart:219:5)
flutter: #38     _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding._invokeFrameCallback (package:flutter/src/scheduler/binding.dart:990:15)
flutter: #39     _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding.handleDrawFrame (package:flutter/src/scheduler/binding.dart:930:9)
flutter: #40     _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding._handleDrawFrame (package:flutter/src/scheduler/binding.dart:842:5)
flutter: #41     _invoke (dart:ui/hooks.dart:154:13)
flutter: #42     _drawFrame (dart:ui/hooks.dart:143:3)
```

## Doctor

```
[✓] Flutter (Channel beta, v0.11.8, on Mac OS X 10.13.6 17G3025, locale es-ES)
    • Flutter version 0.11.8 at /Users/francisco/Code/flutter
    • Framework revision f5b02e3c05 (16 hours ago), 2018-11-20 07:49:29 -0800
    • Engine revision 1baf081343
    • Dart version 2.1.0 (build 2.1.0-dev.9.4 f9ebf21297)

[✓] Android toolchain - develop for Android devices (Android SDK 28.0.3)
    • Android SDK at /Users/francisco/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-28, build-tools 28.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1136-b06)
    • All Android licenses accepted.

[✓] iOS toolchain - develop for iOS devices (Xcode 10.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 10.1, Build version 10B61
    • ios-deploy 1.9.4
    • CocoaPods version 1.5.3

[✓] Android Studio (version 3.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1136-b06)

[✓] IntelliJ IDEA Community Edition (version 2018.2.6)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    • Flutter plugin version 30.0.2
    • Dart plugin version 182.5124

[✓] VS Code (version 1.29.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 2.20.0

[✓] Connected device (3 available)
    • LG D802       • LGD802570857fb                           • android-arm • Android 5.0.2 (API 21)
    • iPhone        • 57a946f59afab5315f585293b4060318c0949f2d • ios         • iOS 11.4.1
    • iPhone 7 Plus • BC555D73-F7F5-4E04-9425-7A2713C73E19     • ios         • iOS 12.1 (simulator)
```

**Comment by zoechi** [comment]
Looks like a dup of #23087 and #13452

**Comment by Eimji** [comment]
Hello same issue here

Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel dev, v0.11.9, on Mac OS X 10.14.1 18B75, locale fr-FR)
[✓] Android toolchain - develop for Android devices (Android SDK 28.0.2)
[✓] iOS toolchain - develop for iOS devices (Xcode 10.1)
[✓] Android Studio (version 3.2)
[✓] Connected device (1 available)

**Comment by Eimji** [comment]
zoechi thanks for your answer.

I have already read your link, but I didn’t inderstand. I need to wait for the Flutter team to fix the issue? Or is there something I can do in my code to overcome the issue?

**Comment by zoechi** [comment]
Yes, you need to wait for the issue to be fixed and updates will be posted to #13452. This issue is closed.
